tests/artifactoryctx_test.go

```diff
@@ -40,8 +39,6 @@ func testCtxTimeout(t *testing.T) {
 	assert.NoError(t, err)
 	time.Sleep(time.Millisecond * 300)
 	_, err = sm.GetVersion()
-	assert.Error(t, err)
-	if !errors.Is(err, context.DeadlineExceeded) {
-		t.Fail()
-	}
+	// Expect timeout error
+	assert.ErrorContains(t, err, context.DeadlineExceeded.Error())
```

Comment:
Are you sure this change is safe?
`errors.Is` os very different than `ErrorContains`.
The former checks the type of the error, and the later checks a string inside the error message.

Reply:
yeah because the new error now is wrapped by a more informative one
@eyalbe4

Reply:
:+1: